pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[project]
+name = "haystack-tutorials"
+dynamic = [
+  "version",
+]
+description = "Tutorials for Haystack: Neural Question Answering & Semantic Search at Scale"
+readme = "README.md"
+license = "Apache-2.0"
+requires-python = ">=3.7"
+authors = [
+  { name = "deepset.ai", email = "malte.pietsch@deepset.ai" },
+]
+
+[project.urls]
+"CI: GitHub" = "https://github.com/deepset-ai/haystack-tutorials/actions"
+"GitHub: issues" = "https://github.com/deepset-ai/haystack-tutorials/issues"
+"GitHub: repo" = "https://github.com/deepset-ai/haystack-tutorials"
+
+[tool.black]
```

Comment:
I think we can keep only this section and remove the `project` ones as this repo won't be distributed as a package and the metadata won't be much useful

Reply:
Ok. Done.